includes/classes/rest/class-rest-plugins-controller.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Class PerformanceLab\REST_API\REST_Plugins_Controller
+ *
+ * @package   PerformanceLab\REST_API
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace PerformanceLab\REST_API;
+
+use PerformanceLab\classes\Plugin_Manager as Plugin_Manager;
+use PerformanceLab\REST_API\REST_Route as REST_Route;
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+
+/**
+ * Class for handling standalone performance plugin actions via REST endpoints.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class REST_Plugins_Controller {
```

Comment:
Do we really need those REST routes? While I agree the REST API is generally the recommended approach over the oldschool AJAX request, since we are working on a WP core feature plugin here and since this functionality is supplemental, we should keep our maintenance burden as small as possible and use what core already provides. Is there a good reason not to use `wp_ajax_install_plugin()` and `WP_REST_Plugins_Controller`?